dev/firewalld_integration_test.sh

```
@@ -0,0 +1,267 @@
+    
+    # Start the services
+    docker compose -f docker-compose.yml down --remove-orphans || true
+    docker compose -f docker-compose.yml up -d --build
+    
+    # Wait for services to be ready
+    info "Waiting for services to start..."
+    retry_count=0
+    max_retries=60
+    retry_interval=2
+
+    until $(curl --output /dev/null --silent --fail "$BASE_URL/health"); do
```

Shellcheck (0.11.0)

[warning] Remove surrounding $() to avoid executing output (or use eval if intentional).

(SC2091)